README.md

```diff
@@ -1,2 +1,10 @@
 # comp110-worksheet-D
 Base repository for COMP110 worksheet D
+
+My implementation of tic-tac-toe. I wasn't sure if created new classes was ok for this assignment so I wrote all functions in oxo.py. Obviously, utility functions like compare_lists and covert_to_1d_index should be in a seperate class.  
```

Comment:
For future reference, adding your own classes is fine if it improves readability